scrapi/base/schemas.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+from copy import deepcopy
+
+from nameparser import HumanName
+
+
+def default_name_parser(names):
+    names = names.split(';')
+
+    contributor_list = []
+    for person in names:
+        name = HumanName(person)
+        contributor = {
+            'prefix': name.title,
+            'given': name.first,
+            'middle': name.middle,
+            'family': name.last,
+            'suffix': name.suffix,
+            'email': '',
+            'ORCID': ''
+        }
+        contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def format_tags(all_tags):
+    tags = []
+    for taglist in all_tags.split(' '):
+        tags += taglist.split(',')
```

Comment:
Yeah, I think it would, though it's hard to know how to split it reliably without doing this